app/common/alerts.py

```diff
@@ -20,17 +20,28 @@ def send(email, reference):
 
 
 class AlertUser:
-    """Alert User"""
+    """Alert goes via gov notify (0) or via logs (1)"""
+
+    NOTIFY_VIA_LOGGING = settings.NOTIFY_VIA_LOGGING
```

Comment:
A more standard way of setting this is probably having this in an `__init__` function